### DISCUSSION TITLE
Big projects

### DISCUSSION THREAD

**Discussion by sbc640964** [livewire/livewire#5492] [1. Help]
Hi, Greetings.
I'm looking to see if there are any complex projects that use livewire

Complex project for data application tables forms and dashboards etc.

**Comment by ariaieboy** [comment]
> Hi, Greetings. I'm looking to see if there are any complex projects that use livewire
> 
> Complex project for data application tables forms and dashboards etc.

Filament is an excellent example of using livewire for data tables and forms etc

it is used in multiple projects. I personally used it on 1 of my medium projects and it's working great
